includes/404-widget.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * LSX functions and definitions - Config.
+ *
+ * @package    lsx
+ * @subpackage 404-widget
+ */
+
+function lsx_widget_area_404_init() {
+	register_sidebar( array(
+			'name'          => esc_html__( '404 page', 'lsx' ),
```

Comment:
@viscosho please run the gulp tasks related to the translation files. As we can see here, you have a new test that is marked as translatable, so we need to make sure the LSX translate files are updated.